package.json

```diff
@@ -44,8 +43,8 @@
     "babel-preset-env": "^1.6.1",
     "babel-preset-react": "^6.5.0",
     "babel-standalone": "^6.7.7",
-    "draft-convert": "^1.3.1",
-    "draft-js": "^0.8.1",
+    "draft-convert": "^2.1.4",
```

Comment:
What're the differences between `draft-convert@1` and `draft-convert@2`?

Reply:
mostly entity API changes - nothing really is affected just figured it was worth also updating